CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [v1.12.0](https://github.com/openebs/cstor-csi/tree/v1.12.0) (2020-07-15)
+
+**Merged pull requests:**
+
+- refact\(operator\): mount host root partition inside container \(\#101\) [\#104](https://github.com/openebs/cstor-csi/pull/104) ([prateekpandey14](https://github.com/prateekpandey14))
```

Comment:
Didn't we revert this commit? still, do we need to mention?

Reply:
Yes, changelog generated before revert, updated now